arches/install/arches-templates/project_name/settings.py-tpl

```diff
@@ -145,10 +145,12 @@ INSTALLED_APPS = (
     "oauth2_provider",
     "django_celery_results",
     # "silk",
-    "{{ project_name }}",
+    "{{ project_name }}",  # Ensure the project is listed before any other arches applications 
```

Comment:
Added this instruction, though I'm wondering if there's a better way to ensure ordering?

Reply:
Nice. Let's see how this goes in practice: we can keep tinkering with the docs without it being a breaking change. I'm thinking it's the responsibility of the readme in the arches application to make this as clear as possible for their users.